On debian testing, python 3.8.3rc1, version parsing fails.

I get the following trace on any use of the library (using latest version from pypi)
```

tabularfile/csv.py:44: in __init__
    self.encoding = charset_normalizer.detect(header).get('encoding')
/tmp/tox-bdauvergne/tabularfile/py3-charsetnormalizer/lib/python3.8/site-packages/charset_normalizer/legacy.py:20: in detect
    r = CnM.from_bytes(byte_str).best().first()
/tmp/tox-bdauvergne/tabularfile/py3-charsetnormalizer/lib/python3.8/site-packages/charset_normalizer/normalizer.py:395: in from_bytes
    py_v = [int(el) for el in python_version_tuple()]
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

.0 = <tuple_iterator object at 0x7f51145cc8e0>

>   py_v = [int(el) for el in python_version_tuple()]
E   ValueError: invalid literal for int() with base 10: '3rc1'
```
Maybe you could simply use `sys.version_info` ?

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,14 +1,16 @@
 language: python
 python:
-  - "2.7"
   - "3.5"
   - "3.6"
   - "3.7"
+  - "3.8"
+  - "3.9"
+  - "3.10-dev"
 
 matrix:
   allow_failures:
-    - python: "2.7"
     - python: "3.5"  # TODO: Investigate why language detection act differently
+    - python: "3.10-dev"
 
 install:
   - pip install nose codecov

--- a/README.md
+++ b/README.md
@@ -1,4 +1,4 @@
-<h1 align="center">Welcome to Charset Detection for Humans 👋 <a href="https://twitter.com/intent/tweet?text=The%20Real%20First%20Universal%20Charset%20%26%20Language%20Detector&url=https://www.github.com/Ousret/charset_normalizer&hashtags=python,encoding,chardet,developers"><img src="https://img.shields.io/twitter/url/http/shields.io.svg?style=social"/></a></h1>
+<h1 align="center">Charset Detection, for everyone 👋 <a href="https://twitter.com/intent/tweet?text=The%20Real%20First%20Universal%20Charset%20%26%20Language%20Detector&url=https://www.github.com/Ousret/charset_normalizer&hashtags=python,encoding,chardet,developers"><img src="https://img.shields.io/twitter/url/http/shields.io.svg?style=social"/></a></h1>
 
 <p align="center">
   <sup>The Real First Universal Charset Detector</sup><br>

--- a/charset_normalizer/normalizer.py
+++ b/charset_normalizer/normalizer.py
@@ -4,7 +4,7 @@ import re
 import statistics
 from encodings.aliases import aliases
 from os.path import basename, splitext
-from platform import python_version_tuple
+from sys import version_info
 from warnings import warn
 
 from cached_property import cached_property
@@ -392,7 +392,7 @@ class CharsetNormalizerMatches:
                 excluded_list=', '.join(cp_exclusion))
 
         # Bellow Python 3.6, Expect dict to not behave the same.
-        py_v = [int(el) for el in python_version_tuple()]
+        py_v = [int(el) for el in (version_info.major, version_info.minor, version_info.micro,)]
         py_need_sort = py_v[0] < 3 or (py_v[0] == 3 and py_v[1] < 6)
 
         supported = collections.OrderedDict(aliases).items() if py_need_sort else aliases.items()

--- a/charset_normalizer/version.py
+++ b/charset_normalizer/version.py
@@ -2,5 +2,5 @@
 Expose version
 """
 
-__version__ = "1.3.4"
+__version__ = "1.3.5"
 VERSION = __version__.split('.')

--- a/setup.py
+++ b/setup.py
@@ -23,11 +23,10 @@ REQUIRES_PYTHON = '>=3.5.0'
 VERSION = get_version()
 
 REQUIRED = [
-    'cached_property',
-    'dragonmapper',
-    'zhon',
-    'prettytable',
-    'loguru'
+    'cached_property>=1.5,<2.0',
+    'dragonmapper>=0.2,<0.3',
+    'prettytable>=1.0,<2.0',
+    'loguru>=0.5,<0.6'
 ]
 
 EXTRAS = {
@@ -80,6 +79,8 @@ setup(
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
         'Topic :: Text Processing :: Linguistic',
         'Topic :: Utilities',
         'Programming Language :: Python :: Implementation :: PyPy'

